apis/theme/src/style-resolver.js

```diff
@@ -56,12 +56,19 @@ function getObject(root, steps) {
 }
 
 function searchPathArray(pathArray, attribute, theme) {
+  const attributeArray = attribute.split('.');
   for (let i = 0; i < pathArray.length; i++) {
     const target = getObject(theme, pathArray[i]);
-    if (target !== null && target[attribute]) {
-      return target[attribute];
+    if (target !== null) {
+      if (attributeArray.length === 1 && target[attribute]) {
+        return target[attribute];
+      }
+      if (attributeArray.length > 1) {
+        return getObject(target, attributeArray);
+      }
```

Comment:
why not just
```suggestion
      return getObject(target, attributeArray);
```
(the result of getObject needs to be changes from null to undefined (when not found) to match the jsdoc)

alternative could be to change
```
const target = getObject(theme, pathArray[i]);
```
to
```
return getObject(theme, [...pathArray[i], ...attributeArray]);
```

Reply:
Awesome! I did an update and added a unit test for it.